### PR TITLE
Nov30 prd release notes

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 1]
```diff
@@ -99,13 +100,17 @@ You can use the [TechPass portal](http://portal.techpass.gov.sg) to do a self-se
   When you scan the QR code on your computer screen, your WOG account gets listed on the authenticator app, and your activation status is confirmed when you click **Next**.
   5. In the **Additional security verification** page, click **Next**.
   <kbd>![after-scan](assets/images/security-verification-for-wog/additional-security-verification-next.png)</kbd>
-  A notification is sent to your mobile app to verify that you are reachable on your mobile device.
-  6. Approve sign-in on the **Authenticator** app.
+
+  A number is shown on your computer screen.
```
Comment (yilunyou) [inline docs/onboard-public-officers-using-non-se-machines.md:104]:
```suggestion
  A number is shown on your browser.
```

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 2]
```diff
@@ -178,9 +180,11 @@ Onboard to TechPass within 30 days of receiving the TechPass invitation email. I
 
   <kbd>![after-scanning-qr-code](assets/images/onboarding/po-non-se/after-scanning-qr-code.png)</kbd>
 
-  The Authenticator sends a notification for you to approve and confirm if the security verification set-up is correct.
+  A number is shown on your computer screen.
```
Comment (yilunyou) [inline docs/onboard-public-officers-using-non-se-machines.md:183]:
```suggestion
  A number is shown on your browser.
```

File: docs/onboard-vendors-to-techpass.md [hunk 3]
```diff
@@ -87,15 +87,15 @@ Your project manager or reporting officer will work with the engaging government
   4. Go back to your computer and click **Next**.
   <kbd>![vendor-mfa-2](assets/support/vendor-mfa-2.png)</kbd>
   5. Scan the QR code on your computer screen and click **Next**. Your TechPass account gets activated and linked to the authenticator app.
-    <kbd>![vendor-scan-qr-code](assets/support/vendor-mfa-3.png)</kbd>
+    <kbd>![vendor-scan-qr-code](assets/support/vendor-mfa-3.png)</kbd>     
+  
+  A number is shown on your computer screen.
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:92]:
```suggestion
  A number is shown on your browser.
```

File: docs/request-for-seed-provisioning.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Request for SEED provisioning using TechPass portal
+
+The following table tells who can complete the steps mentioned in this article to request for SEED provisioning using the TechPass portal.
+
+ | User  | Device | Active TechPass | TechPass ID's domain format | Request for SEED provisioning |
+| ------------- |:-------------| :------------- | :------------- | ------ |
+| Public officer      | non-SE GSIB | Yes | *\<acronym-for-your-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*.| [TechPass portal](https://portal.techpass.gov.sg) |
+| Public officer | SE GSIB | Yes| *\<acronym-for-your-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*. | Create a [service request](https://go.gov.sg/techpass-sr) |
+| Vendor | non-SE GSIB | Yes | *\<acronym-for-your-engaging-agency\>.gov.sg*. <br><br>For example, *tech.gov.sg*.| [TechPass portal](https://portal.techpass.gov.sg) |
```
Comment (yilunyou) [inline docs/request-for-seed-provisioning.md:9]: this category isn't exactly right.. can we leave it out for now?

File: docs/request-for-seed-provisioning.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+*To request for SEED provisioning*
+
+1. From your non-SE GSIB device, go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
+2. Sign in to your TechPass account.
+<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+
+> **Note**
+> If you are a public officer you may be prompted to log in to the WOG account first before proceeding to TechPass account.
+
+You will now be prompted to approve your TechPass sign-in. A number will be shown on your computer.
```
Comment (yilunyou) [inline docs/request-for-seed-provisioning.md:21]:
```suggestion
You will now be prompted to approve your TechPass sign-in. A number will be shown on your browser.
```

File: docs/whats-new/staging-release-notes.md [hunk 6]
```diff
@@ -228,7 +228,7 @@ TechPass automatically terminates TechPass accounts that have not been used with
 
 Log in with your TechPass and complete the TechPass onboarding flow.
 
-- If you are a public officer, [accept the invitation and complete the onbaording flow](/onboard-public-officers-using-non-se-machines?id=step-3-accept-invitation).
+- If you are a public officer, [accept the invitation and complete the onbaording flow](/onboard-public-officers-using-non-se-devices?id=step-3-accept-invitation).
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:231]:
```suggestion
- If you are a public officer, [accept the invitation and complete the onboarding flow](/onboard-public-officers-using-non-se-devices?id=step-3-accept-invitation).
```

File: docs/whats-new/production-release-notes.md [hunk 7]
```diff
@@ -226,7 +226,7 @@ TechPass automatically terminates TechPass accounts that have not been used with
 
 Log in with your TechPass and complete the TechPass onboarding flow.
 
-- If you are a public officer, [accept the invitation and complete the onbaording flow](/onboard-public-officers-using-non-se-machines?id=step-3-accept-invitation).
+- If you are a public officer, [accept the invitation and complete the onbaording flow](/onboard-public-officers-using-non-se-devices?id=step-3-accept-invitation).
```
Comment (yilunyou) [inline docs/whats-new/production-release-notes.md:229]:
```suggestion
- If you are a public officer, [accept the invitation and complete the onboarding flow](/onboard-public-officers-using-non-se-devices?id=step-3-accept-invitation).
```

File: docs/request-for-seed-provisioning.md (hunk 4 above)
Comment (SenthilKumarSivakami) [reply docs/request-for-seed-provisioning.md:9]: so can i just say any user with an active TP and using a non-SE GSIB device can follow the steps to request for SEED provisioning?

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**